client/state/data-layer/wpcom/me/devices/test/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { NOTICE_CREATE, USER_DEVICES_ADD } from 'state/action-types';
+import {
+	requestUserDevices,
+	handleSuccess,
+	handleError,
+} from '../';
+
+describe( 'wpcom-api', () => {
+	describe( 'user devices', () => {
+		describe( '#requestUserDevices', () => {
+			it( 'should dispatch HTTP request to the users devices endpoint', () => {
+				const action = { type: 'DUMMY' };
+				const dispatch = spy();
+				const next = spy();
+
+				requestUserDevices( { dispatch }, action, next );
```

Comment:
changes are occurring in the middleware, you may want to remove the spy for `next()` and replace it with `null` here